client/src/pages/AboutPage/AboutPage.jsx

```diff
@@ -0,0 +1,51 @@
+import "./AboutPage.css";
+
+function AboutPage() {
+  return (
+    <div className="about-wrapper">
+      <div className="about-container">
+        <h1 className="about-title">À Propos</h1>
+        <div className="about-content">
+          <p>
+            <strong>Bienvenue chez Eating Nam Mam!</strong>
+            <br />
```

Comment:
Instead of a "strong" tag inside of a "p" tag, use "h2" for the subtitle and then a "p" tag. CSS will do the trick to insert space between both elements so you can delete the "br" one. Do the same for all the paragraphs.

Reply:
Done

Reply:
Done